packages/components/src/Banner/Banner.module.css

```diff
@@ -66,18 +62,10 @@
   margin-bottom: 0;
 }
 
-.medium .bannerChildren {
-  flex: 1;
```

Comment:
I _think_ this line might be redundant with L61, as the earlier declaration would apply to `.bannerChildren` with and without `.medium` applied